src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -96,16 +97,12 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_POLICY: '2018-05-01',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2018-05-01',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
-        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-01-01-preview', {
-            'classic_administrators': '2015-06-01'
-        }),
-        ResourceType.DATA_STORAGE: '2018-03-28',
-        ResourceType.DATA_COSMOS_TABLE: '2017-04-17',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
         ResourceType.MGMT_KEYVAULT: '2018-02-14',
         ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-01-01-preview', {
             'classic_administrators': '2015-06-01'
         }),
+        ResourceType.MGMT_CONTAINERREGISTRY: '2018-09-01',
```

Comment:
https://github.com/Azure/azure-rest-api-specs/tree/master/profile

ACR isn't part of either of these profiles. It should only appear in latest. Does it fail if you leave these out?

Reply:
The CI was complaining couldn't find acr from these profiles.

Reply:
Hmmm. Okay. I will create an issue for us to look into this.